HLTrigger/Configuration/test/cmsDriver.csh

```diff
@@ -13,13 +13,13 @@ rehash
 #
 # gen sim input files for Monte-Carlo tests
 #   InputGenSimGRun0 = /store/relval/CMSSW_8_0_11/RelValProdTTbar/GEN-SIM/80X_mcRun1_realistic_v4-v1/10000/06A6C86B-C634-E611-93A5-0CC47A74525A.root
-set InputGenSimGRun0 = /store/group/dpg_trigger/comm_trigger/TriggerStudiesGroup/STORM/GEN-SIM/CMSSW_8/06A6C86B-C634-E611-93A5-0CC47A74525A.root
+set InputGenSimGRun0 = root://eoscms.cern.ch//eos/cms/store/group/dpg_trigger/comm_trigger/TriggerStudiesGroup/STORM/GEN-SIM/CMSSW_8/06A6C86B-C634-E611-93A5-0CC47A74525A.root
```

Comment:
Yes, it should be fine and usually it is.
The problem is that sometimes using /store/group/..../ outside CERN gave me problems or was slow. 
Adding root://eoscms.cern.ch//eos/cms/ as prefix solved that problem. This change is fully compatible and is faster because it don't need anymore to look where the file is.